test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -147,6 +161,37 @@ class AccountListPage {
     );
   }
 
+  /**
+   * Adds a new BTC account with default next available name.
+   *
+   * @param btcAccountCreationEnabled - indicates if the BTC account creation is expected to be enabled or disabled. Defaults to true.
+   */
+  async addNewBtcAccountWithDefaultName(
```

Comment:
Maybe we can make this function a bit more generic, so in case we need to add a test with a btc account with a custom name we can re-use the same function, and we don't need to update all specs/places using this function

```
async addNewBtcAccount({
   btcAccountCreationEnabled: boolean = true,
   accountName: string = ''
}
....
// if accountName is empty, use default flow -- and we can leave the other flow, since it's not used anywhere now, but it will help grow this function in the future
```

what do you think?

Reply:
thanks for the suggestion, addressed in [0719d7b](https://github.com/MetaMask/metamask-extension/pull/28437/commits/0719d7bc549036f37dd3aa7a1a08318808760b9e)